easybuild/easyconfigs/m/MaCH/MaCH-1.0.18.c-ictce-6.2.5.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
```

Comment:
minor changes needed here, `easyblock = 'ConfigureMake'` , put this in REVIEW plz!